### PR TITLE
test/169: add tests

### PR DESCRIPTION
**Pull request by NMarteniuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    private readonly Mock<ILoggerService> _mockLogger;
+    public UpdateCategoryContentHandlerTests()
+    {
+        _mockRepositoryWrapper = new Mock<IRepositoryWrapper>();
+        _mockMapper = new Mock<IMapper>();
+        _mockLogger = new Mock<ILoggerService>();
+    }
+
+    [Fact]
+    public async void Handle_Should_ReturnSuccess_WhenSaveChangesAsyncWorkedCorrect()
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs:35]: Task

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+using Xunit;
+using Streetcode.BLL.MediatR.Sources.SourceLinkCategory.Update;
+using FluentAssertions;
+using FluentResults;
+
+namespace Streetcode.XUnitTest.MediatRTests.Sources;
+
+public class UpdateCategoryContentHandlerTests
+{
+    private const int SUCCESSFULRESULT = 1;
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs:21]: error in name - SUCCESSFULRESULT

**Review by yaninavoda — APPROVED** [review]
ok!

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs (hunk 2 above)
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs:21]: add private const int EXISTS_STREETCODE_ID = 1;
private const int NOTEXISTS_STREETCODE_ID = -1; use below

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+        // Assert
+        result.Errors.Should().ContainSingle(e => e.Message == UpdateFailedErrorMessage(request));
+    }
+
+    private void SetupMockRepository(CategoryContentUpdateDto request, int resultCase)
+    {
+        var streetcode = request.StreetcodeId switch
+        {
+            1 => new StreetcodeContent { Id = request.StreetcodeId },
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Sources/UpdateCategoryContentHandlerTests.cs:140]: use const EXISTS_STREETCODE_ID instead magic number

**Review by OlhaPivniuk — COMMENTED** [review]
Added comments

**Review by OlhaPivniuk — APPROVED** [review]

**Pull request merged**